### ISSUE TITLE
using a Pathname as file argument results in error.

### ISSUE THREAD

**Issue by rempargo** [mceachen/exiftool#8]
When using a Pathname instead of a String for the file argument, I get an error.
See the following code:

```
require 'exiftool'
require 'pathname'
path=Pathname.new('p./public/images/test.jpg')
Exiftool.new(path)
```

will result in 
`NoMethodError: undefined method empty?' for #<Pathname:p./public/images/test.jpg>`

the workaround is to convert the Pathname to a String:
`Exiftool.new(path.to_s)`

_Ruby version 2.3.1_

**Comment by mceachen** [comment]
pull requests are welcome (and this, specifically, seems fine)

**Comment by mceachen** [comment]
I just released v1.1.0 which addresses this.
